src/erasure-code/jerasure/jerasure_init.cc

```diff
@@ -3,7 +3,7 @@
 /*
  * Ceph distributed storage system
  *
- * Copyright (C) 2014 Cloudwatt <libre.licensing@cloudwatt.com>
```

Comment:
I changed jerasure_init to take an array of words to use and its invoked from jerasure and shec.